content/en/agent/fleet_automation/setup.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Set up Fleet Automation
+disable_toc: false
+further_reading:
+- link: "/remote_configuration"
+  tag: "Documentation"
+  text: "Find out more about Remote Configuration"
+- link: "/infrastructure/list/#agent-configuration"
+  tag: "Documentation"
+  text: "Learn about the Agent configuration view"
+- link: "https://www.datadoghq.com/blog/fleet-automation/"
+  tag: "Blog"
+  text: "Centrally govern and remotely manage Datadog Agents at scale with Fleet Automation"
+---
+
+To ensure you have access to all Fleet Automation features, upgrade your Agents to version 7.47.0 or later and follow the instructions below to set up Remote Configuration for your Agents.
+
+## Remote Configuration
+
+To use Fleet Automation, you must enable Remote Configuration.
+
+### Prerequisites
+
+- Datadog Agent version `7.47.0`
+- Ensure your RBAC permissions include [`org_management`][1], so you can enable Remote Configuration for your organization.
+- Ensure your RBAC permissions include [`api_keys_write`][2], so you can create a new API key with the Remote Configuration capability, or add the capability to an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. A key with this capability allows you to authenticate and authorize your Agent to use Remote Configuration.
+
+Datadog recommends upgrading your Agents regularly to make sure you have access to the latest features.
+
+### Setup
+
+1. Ensure that [Remote Configuration is enabled for your organization][3].
+1. Select an existing API key or create a new API key, and enable the Remote Configuration capability on the key if it is not already enabled.
+
+   {{<img src="agent/remote_config/RC_Key_updated.png" alt="API Key properties with Remote Configuration capability Enable button." width="90%" style="center">}}
+1. Restart your Agent.
+
+### Agent Remote Configuration status
+
+Gain visibility into the Remote Configuration status of your Agent through the [Remote Configuration UI][3].
```

Comment:
Isn't it better to send them to Fleet directly?

Reply:
I've moved this page to `agent/guide/setup_remote_config`. I might be mistaken, but I think the information covered in the table is only available from the Remote Config UI. That's the only place I could find it.